internal/provider/provider.go

```diff
@@ -531,6 +532,12 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData, p *schema.Pr
 	}
 	tflog.Info(ctx, fmt.Sprintf("Provider: acceptance test mode is %t\n", acceptanceTestMode))
 
+	liveProductionTestMode := false
+	if os.Getenv("TF_ACC_PROD") == "1" {
+		liveProductionTestMode = true
+	}
+	tflog.Info(ctx, fmt.Sprintf("Provider: live production test mode is %t\n", liveProductionTestMode))
```

Comment:
nit: Is it a valid scenario for both of these test flags to be enabled at the same time? If not, we should probably add a quick check here to throw an error. That could really save us if someone accidentally messes up the environment variables and defaults all tests to the acceptance/live thresholds.